src/components/signup/Signup.html

```diff
@@ -0,0 +1,87 @@
+            <div class="input-email">
+              <label class="label-font" for="email-tag">이메일</label>
+              <div class="input-content">
+                <input class="email-tag" autocomplete="off" id="email-tag" type="email" placeholder="이메일을 입력해주세요"
+                  required>
+                <div
+                  style="margin-left: 12px; margin-right: 24px; width: 24px; height: 24px; display: flex; justify-content: center; align-items: center;">
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="input-nickname">
+            <label class="label-font" for="nickname-tag">닉네임</label>
+            <div class="input-content">
+              <input class="nickname-tag" autocomplete="off" id="nickname-tag" type="text" placeholder="닉네임을 입력해주세요"
+                required>
+              <div
+                style="margin-left: 12px; margin-right: 24px; width: 24px; height: 24px; display: flex; justify-content: center; align-items: center;">
+              </div>
+            </div>
+          </div>
+          <div class="input-password">
+            <label class="label-font" for="password-tag">비밀번호</label>
+            <div class="input-content">
+              <input class="password-tag" autocomplete="off" id="password-tag" type="password"
+                placeholder="비밀번호를 입력해주세요" required>
+              <img
+                style="margin-left: 12px; margin-right: 24px; width: 24px; height: 24px; display: flex; justify-content: center; align-items: center;"
+                class="password-icon" src="/src/assets/img/btn_visibility_off.svg">
+            </div>
+          </div>
+          <div class="input-password-check">
+            <label class="label-font" for="password-check-tag">비밀번호 확인</label>
+            <div class="input-content">
+              <input class="password-check-tag" autocomplete="off" id="password-check-tag" type="password"
+                placeholder="비밀번호를 입력해주세요" required>
+              <img
+                style="margin-left: 12px; margin-right: 24px; width: 24px; height: 24px; display: flex; justify-content: center; align-items: center;"
+                class="password-check-icon" src="/src/assets/img/btn_visibility_off.svg">
+            </div>
+          </div>
+        </div>
+        <button class="login-submit-btn btn is-cursor-pointer" type="submit" disabled>회원가입</button>
+      </div>
+      <div class="convenience-login">
+        <div class="convenience-login-content">
+          <p>간편 로그인하기</p>
+          <nav>
+            <ul>
+              <a href="https://www.google.com/" target="_blank" rel="noopener noreferrer">
+                <li
+                  style="display: flex; justify-content: center; align-items: center; width: 42px; height: 42px; background: white; border-radius: 9999px; border: 1px #F9FAFB">
+                  <img src="/src/assets/img/ic_google.png" alt="구글 로그인">
+                </li>
+              </a>
```

Comment:
P3:
unordered list 의 list item 태그인 li는 ul의 바로 자식으로 있는 것이 더 가독성에 좋을 것 같습니다.

```suggestion
              <li
                  style="display: flex; justify-content: center; align-items: center; width: 42px; height: 42px; background: white; border-radius: 9999px; border: 1px #F9FAFB">
                <a href="https://www.google.com/" target="_blank" rel="noopener noreferrer">
                  <img src="/src/assets/img/ic_google.png" alt="구글 로그인">
                </a>
              </li>
```